src/Core/Table/ArcTable.fs

```diff
@@ -468,105 +494,107 @@ type ArcTable(name: string, headers: ResizeArray<CompositeHeader>, values: Syste
             newTable
 
     // - Row API - //
-    member this.AddRow (?cells: CompositeCell [], ?index: int) : unit = 
+    member this.AddRow (?cells: ResizeArray<CompositeCell>, ?index: int) : unit =
         let index = defaultArg index this.RowCount
-        let cells = 
+        let cells =
```

Comment:
Again this behaves different then my expecatation. Did we not decide on a sparse approach?

Reply:
Will wait for your answers before continuing the review.

Reply:
True that, this should just increase the rowCount of the "Values"